modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -1896,7 +1896,7 @@ export class BitGoAPI implements BitGoBase {
     return _.merge({}, defaultConstants(this.getEnv()), BitGoAPI._constants[this.getEnv()]);
   }
 
-  async getBitgoChallengesForEcdsaSigning(walletId: string): Promise<any> {
+  async getChallengesForEcdsaSigning(walletId: string): Promise<any> {
```

Comment:
The name was a bit misleading before, since it fetches both enterprise and bitgo challenges.